test/unittest/memory/memory_planner_validate.cpp

```diff
@@ -0,0 +1,173 @@
+// SPDX-License-Identifier: Apache-2.0
+/**
+ * Copyright (C) 2021 Parichay Kapoor <pk.kapoor@samsung.com>
+ *
+ * @file   memory_planner_validate.h
+ * @date   11 August 2021
+ * @see    https://github.com/nnstreamer/nntrainer
+ * @author Parichay Kapoor <pk.kapoor@samsung.com>
+ * @bug    No known bugs except for NYI items
+ * @brief  Tests for memory planning
+ */
+
+#include <bitset>
+#include <random>
+#include <vector>
+
+#include <basic_planner.h>
+#include <memory_planner_validate.h>
+
+constexpr unsigned int MEM_BYTES = 50;
+constexpr unsigned int MEM_QUANT = 100;
+constexpr unsigned int INTERVAL_SIZE = 5;
+
+void MemoryPlannerValidate::SetUp() {
+  plan_type = GetParam();
+
+  planner = nullptr;
+  if (plan_type == nntrainer::BasicPlanner::type)
+    planner = std::move(std::make_unique<nntrainer::BasicPlanner>());
```

Comment:
It looks like given value is already rvalue so no need to std::move.

Reply:
done.